lib/features/manage_account/presentation/manage_account_dashboard_view.dart

```diff
@@ -146,6 +147,8 @@ class ManageAccountDashBoardView extends GetWidget<ManageAccountDashBoardControl
           return ProfilesView();
         case AccountMenuItem.languageAndRegion:
           return LanguageAndRegionView();
+        case AccountMenuItem.emailRules:
+          return EmailRulesView();
```

Comment:
please check the session to show/or not show this item. It is an extension of Linagora. It means that other JMAP server may not include it.